packages/react-link/src/components/Link/useLinkState.ts

```diff
@@ -1,62 +1,28 @@
 import * as React from 'react';
-import { Enter, Space } from '@fluentui/keyboard-keys';
+import { getCode, EnterKey, SpacebarKey } from '@fluentui/keyboard-key';
 import type { LinkState } from './Link.types';
 
 /**
- * The useLink hook processes the Link draft state.
+ * The useLink hook processes the Link draft state.root.
```

Comment:
Find/replace mistake?
```suggestion
 * The useLink hook processes the Link draft state.
```